src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -52,23 +52,41 @@ const defaultProps = {
     },
 };
 
+function FormSpace() {
```

Comment:
Consider encapsulating FormSpace in the AddressPage, as it is only used in one component.
Or just use <View style={styles.formSpaceVertical} /> instead.